src/core/hle/service/am/am.cpp

```diff
@@ -418,6 +429,94 @@ InstallStatus InstallCIA(const std::string& path,
     return InstallStatus::ErrorInvalid;
 }
 
+InstallStatus InstallFromNus(u64 title_id, int version) {
+#ifdef ENABLE_WEB_SERVICE
+    LOG_DEBUG(Service_AM, "Downloading {:X}", title_id);
+
+    Service::AM::CIAFile installFile(Service::AM::GetTitleMediaType(title_id));
```

Comment:
```suggestion
    Service::AM::CIAFile install_file = Service::AM::GetTitleMediaType(title_id);
```
Also, aren't we in the Service::AM namespace already anyway?